exercises/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,27 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number1) {
+  let result = number1 < 0;
+  return result;
+}
+
+function isBetween5and10(number1) {
+  let result = number1 <= 10 && number1 > 5;
+  return result;
```

Comment:
You can shorten your code by using this "function isNegative(number) {return number < 0 ; } 
and it is same for others.

Reply:
Hi Yasamin, Thank you for the feedbacks, You mean not to assign it to a variable result? If yes, you are right there isn't any need for the result variable just find this a bit more tidy. Please let me know if you meant something else. Thank you  :)

Reply:
In this case I would just use `return number1 < 0`.

Declaring variables in programming takes up some of your computer's memory when your program is run. In this scenario, the equation is simple enough that I don't think you need to assign it to a variable for clarity first.

Reply:
Great to know, Thank you both, I've change it